src/App.jsx

```diff
@@ -14,33 +16,96 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPrapedGoods(goods, sortField, reversed) {
+  let prepearedGoods = [...goods];
```

Comment:
There is a typo in the variable name `prepearedGoods`. It should be `preparedGoods` to maintain consistency and readability.